src/views/Dashboard.vue

```diff
@@ -37,6 +37,11 @@
                 <h2 class="mb-3" data-test-id="dashboard-filing-history-subtitle">
                   <span>Recent Filing History</span>&nbsp;<span class="gray6">({{historyCount}})</span>
                 </h2>
+                <staff-notation
+                  v-if="visibleForStaff"
```

Comment:
I'd be OK with you using `isRoleStaff` here directly.

Also see my comment in "visibleForStaff".

Reply:
yes, I had some issues with the tests. They showed messages that suggested me I should have a computed property then added it as a test. I removed now. Thanks!